### ISSUE TITLE
API Usage Requires Additional Encrypted Data Submission Beyond Basic Points 

### ISSUE THREAD

**Issue by DacHoaGit** [kaimarks9/moonbix-bot#17]
I noticed that their APIs don't include headers["Authorization"] in the request headers. Additionally, for their url_claim API, besides submitting the score, it also requires submitting some other encrypted data. I believe this data could be tracking the game's coordinates, so the process is more complex than what your code currently implements

**Comment by mz100814** [comment]
bro can you tell me ，how to find Authorization token？

**Comment by Bach0409** [comment]
You can refer to the following source code: https://github.com/Bach0409/MoonBix

**Comment by chiconghvan** [comment]
> You can refer to the following source code: https://github.com/Bach0409/MoonBix

You didn't push your code

**Comment by Bach0409** [comment]
> > You can refer to the following source code: https://github.com/Bach0409/MoonBix
> 
> You didn't push your code

U can ping for me : https://www.facebook.com/bach0409/

**Comment by chiconghvan** [comment]
> > > You can refer to the following source code: https://github.com/Bach0409/MoonBix
> 
> > 
> 
> > You didn't push your code
> 
> 
> 
> U can ping for me : https://www.facebook.com/bach0409/

Just inbox ur fb

**Comment by slavon141** [comment]
Traceback (most recent call last):
  File "D:\moonbix-bot\bot.py", line 8, in <module>
    import requests_html
  File "D:\moonbix-bot\venv\lib\site-packages\requests_html.py", line 14, in <module>
    from lxml.html.clean import Cleaner
  File "D:\moonbix-bot\venv\lib\site-packages\lxml\html\clean.py", line 18, in <module>
    raise ImportError(
ImportError: lxml.html.clean module is now a separate project lxml_html_clean.
Install lxml[html_clean] or lxml_html_clean directly.

**Comment by bachdx040920** [comment]
> > I noticed that their APIs don't include headers["Authorization"] in the request headers. Additionally, for their url_claim API, besides submitting the score, it also requires submitting some other encrypted data. I believe this data could be tracking the game's coordinates, so the process is more complex than what your code currently implements
> 
> exactly.. I feel there are some things missing to run this script and the encrypted data is very difficult to break. if you can code python or your friends, you can recode what I have made. and of course you will get a fair fee and contribution to this project

If u want complete this project, ping for me !!!

**Comment by hasan1818666891** [comment]
Most complex Aes encryption.  : {

**Comment by Bach0409** [comment]
> Most complex Aes encryption. : {

I have done this work, you can contact me to use the tool for this

**Comment by bachdx040920** [comment]
> Most complex Aes encryption. : {

I have completed this work, you can contact me to use the tool for this

**Comment by bachdx040920** [comment]
> > > Most complex Aes encryption. : {
> > 
> > 
> > I have completed this work, you can contact me to use the tool for this
> 
> are u selling ur work?

Yes, if you are interested you can leave your contact telegram, I will contact you

**Comment by bachdx040920** [comment]
> > > > > Most complex Aes encryption. : {
> > > > 
> > > > 
> > > > I have completed this work, you can contact me to use the tool for this
> > > 
> > > 
> > > are u selling ur work?
> > 
> > 
> > Yes, if you are interested you can leave your contact telegram, I will contact you
> 
> sorry mate im not interest.. u can sell to anybody except me.. i try to fix this by myself..
> 
> thank you!

Time is money. Good luck!
